configure.rs

```diff
@@ -0,0 +1,84 @@
+use std::env;
+
+#[allow(dead_code)]
+pub(crate) struct Target {
```

Comment:
Everything here can just be `pub` rather than `pub(crate)`, this isn't user-facing

Reply:
Done.